core/validation/messages.py

```diff
@@ -34,8 +34,11 @@
     "You’ve entered your own content, instead of selecting from the dropdown list provided. "
     "Select an option from the dropdown list."
 )
-OVERSPEND_PROJECT = (
-    "The grand total amount is greater than your allocation. Check the data for each financial year is correct."
+OVERSPEND_CDEL = (
+    "The total CDEL amount is greater than your allocation. Check the data for each financial year is correct."
+)
+OVERSPEND_RDEL = (
+    "The total RDEL amount is greater than your allocation. Check the data for each financial year is correct."
```

Comment:
There's no need for these two messages. I think we can just have ```OVERSPEND``` as an fstring and swap in and out RDEL/CDEL